label_studio/tasks/serializers.py

```diff
@@ -300,14 +300,19 @@ def create(self, validated_data):
             max_inner_id = (tasks.order_by("-inner_id")[0].inner_id + 1) if tasks else 1
 
             for i, task in enumerate(validated_tasks):
+                cancelled_annotations = len([ann for ann in task_annotations[i] if ann.get('was_cancelled', False)])
```

Comment:
Why False? Annotation is cancelled when `was_cancelled == True`, I think.

Reply:
The condition is used if no was_cancelled field found in annotation